cmd/vc-rest/startcmd/start.go

```diff
@@ -97,6 +97,10 @@ const (
 		" Alternatively, this can be set with the following environment variable: " + tlsCACertsEnvKey
 	tlsCACertsEnvKey = "VC_REST_TLS_CACERTS"
 
+	universalRegistrarURLFlagName  = "universal-registrar-url"
```

Comment:
Are we going to make http resolver update based DIDs supported by uniregistrar?

https://github.com/trustbloc/edge-service/blob/7c70ea7d41a469bea12cba110e9d87ec8437e5a3/cmd/vc-rest/startcmd/start.go#L401

Remember, to validate VC we need resolver to allow those DIDs.